### DISCUSSION TITLE
Toggle on/off parts of the 3D model. How to find the part?

### DISCUSSION THREAD

**Discussion by wawa2016** [google/model-viewer#1986] [Q&A] [answered]
When using [Sketchfab Configurator Studio](https://labs.sketchfab.com/experiments/configurator-studio/) There is an option to select in the viewer a specific part of the model and then create on/off visibility toggle for it. Is such feature planned for Model Viewer Editor? Or can I somehow achieve it myself?
![image](https://user-images.githubusercontent.com/16106396/106289857-0c654f00-624a-11eb-8c9f-f690ed87918f.png)

**Comment by elalish** [comment, marked as answer]
Yes, but first we'll need the capability in our scene-graph API: https://github.com/google/model-viewer/discussions/1817

Contributions welcome! This may take awhile otherwise, as we have a few higher priority things to implement first.
